lib/roo_on_rails/rack/populate_env_from_jwt.rb

```diff
@@ -0,0 +1,98 @@
+require 'json/jwt'
+require 'faraday'
+require 'faraday_middleware'
+
+module RooOnRails
+  module Rack
+    class PopulateEnvFromJWT
+      UnnacceptableKeyError = Class.new(RuntimeError)
+      VALID_JWK_URL_PREFIXES = [
+        'https://www.deliveroo.com/identity-keys/',
+        'https://identity.deliveroo.com/identity-keys/'
+      ].freeze
+
+      def initialize(app, logger:)
+        @app = app
+        @keys = {}
+        @logger = logger
+
+        if development?
+          @logger.warn "JWTs will not have their signatures checked, you're in development mode."
```

Comment:
This seems odd... wouldn't we want to check signatures even in development mode?

Reply:
Then you'd have to have a running orderweb/identity in order to run your service (plus we'd have to add `https://identity.deliveroo.com.dev` as a prefix, and point out that this service _needs_ to be at that location for the service you're developing to run).

This also means that there's a much more limited risk of putting an insecure key in the codebase to make development easier and having that key be left in for production 😱

Reply:
Fair enough. Maybe want to make it optional that you can check sigs in dev though in case you want to test it? Even if the default is not to?